### PR TITLE
Add apis to work with translation strings.

### PR DESCRIPTION
**Pull request by bkearney**
Add bindings to
- List out all translations
- List a single translation
- Set a single translation

These are outlined at http://docs.transifex.com/api/translation_strings/

Additionaly, there are whitespace changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,5 +87,16 @@ To download the translations and store them in a local file run the following:
...
 
+#### Working with translations directly
+To list all the translations via the api:
+    In [14]: t.list_translation_strings('helloworld5', 'resourceName', 'pt-br')
+
+To get a single translation
+    In [15]: t.get_translation_string('helloworld5', 'resourceName', 'pt-br', "Some untranslated text")
+
+To set a single translation
+    In [16]: t.get_translation_string('helloworld5', 'resourceName', 'pt-br', "Some untranslated text", some translated text")
```
Comment (jakul) [inline README.md]: @bkearney Is this line correct?

**Comment by jakul** [comment]
Other than the little error in the `README` this looks good to me. Update the `README` and I'll merge

**Comment by jakul** [comment]
Thanks for making this PR @bkearney

**Comment by bkearney** [comment]
@jakul Thanks for the catch. I changed the doc, amended the commit, and pushed a new version.

**Pull request merged**